addons/l10n_ch/models/res_config_settings.py

```diff
@@ -18,3 +18,4 @@ class ResConfigSettings(models.TransientModel):
         related="company_id.l10n_ch_isr_scan_line_left", readonly=False)
     l10n_ch_isr_scan_line_top = fields.Float(string='Vertical offset',
         related="company_id.l10n_ch_isr_scan_line_top", readonly=False)
+    l10n_ch_print_qrcode = fields.Boolean("Print Swiss QR Code", config_parameter='l10n_ch.print_qrcode')
```

Comment:
Nope... Although this is linked to an `ir.config_parameter`, this is still adding a field to `res.config.settings`.

Reply:
@rco-odoo fields of res.config.settings that have the attribute config_parameter=True could be computed/not stored, if you ask me....